README.md

```diff
@@ -1,31 +1,69 @@
 # translator_extensions
+
 The Translator-specific metadata extensions used in the SmartAPI registry
 
 ## x-translator
 #### Status
 This extension's schema has been written, and registry files currently contain this extension.
 
 #### Description
-This extension is inside the OpenAPI info object and contains basic API-level metadata. Currently...
+This extension is inside the OpenAPI info object and contains basic API-level metadata. Currently, there are two (required) properties: component (KP, ARA, ARS, or Utility) and team (one or more Translator team names). Current tags include:
+
+- **infores:** Information Resource CURIE identifying the Translator Resource
+- **component:** One of  "KP", "ARA", "ARS", "Utility"
+- **team**: List of contributing Translator teams (see [schema file](https://github.com/NCATSTranslator/translator_extensions/blob/main/x-translator/smartapi_x-translator_schema.json) for the full list)
+- **biolink-version:**  One SemVer-formatted version number (e.g. '1.8.2', '2.2.16', etc.)
```

Comment:
Give more detail. "This is the biolink-model version that the Translator Resource is currently compliant with / was built to...and this is where you go to look at biolink-model and its versions / releases."